doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -117,11 +164,23 @@ If monit service is not avalaible, will consider system in fault condition.
 FAN/PSU/ASIC data not available will also considered as fault conditon.
 Incomplete data in the DB will also be considered as fault condition, e.g., PSU voltage data is there but threshold data not available.
 
-Monit, thermalctld and psud will raise syslog when fault condition encounterd, so system health monitor will only generate some general syslog on these situation to avoid redundant. For example, when fault condition meet, "system health status change to fault" can be print out, "system health status change to normal" when it recovered.
+Monit, thermalctld and psud will raise syslog when fault condition encountered, so system health monitor will only generate some general syslog on these situation to avoid redundant. For example, when fault condition meet, "system health status change to fault" can be print out, "system health status change to normal" when it recovered.
+
+this service will be started after system boot up(after database.service and updategraph.service).
+
+## 3. System health data in redis database
 
-this service will be started after system boot up(after rc.local).
+System health service will populate system health data to STATE db. A new table "SYSTEM_HEALTH_INFO" will be created to STATE db.
+
+	; Defines information for a system health
+	key                     = SYSTEM_HEALTH_INFO             ; health information for the switch 
+	; field                 = value
+	summary                 = STRING                         ; summary status for the switch
+	<item_name>             = STRING                         ; an entry for a service or device
 
-## 3. Platform API and PMON related change to support this new service
+We store items to db only if it is abnormal.
```

Comment:
can you add an example DB info table here?

Reply:
sure